ddtrace/contrib/pynamodb/patch.py

```diff
@@ -0,0 +1,80 @@
+"""
+Trace queries to botocore api done via a pynamodb client
+"""
+
+import os
+from ddtrace.vendor import wrapt
+from ddtrace import config
+import pynamodb.connection.base
+
+
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY, SPAN_MEASURED_KEY
+from ...pin import Pin
+from ...ext import SpanTypes
+from ...utils.formats import deep_getattr
+from ...utils.wrappers import unwrap
+
+# Pynamodb connection class
+_PynamoDB_client = pynamodb.connection.base.Connection
+
+config._add("pynamodb", {"service_name": os.getenv("DD_PYNAMODB_SERVICE") or "pynamodb",})
+
+
+def patch():
+    if getattr(pynamodb.connection.base, "_datadog_patch", False):
+        return
+    setattr(pynamodb.connection.base, "_datadog_patch", True)
+
+    wrapt.wrap_function_wrapper("pynamodb.connection.base", "Connection._make_api_call", patched_api_call)
+    Pin(service=None).onto(pynamodb.connection.base.Connection)
+
+
+def unpatch():
+    if getattr(pynamodb.connection.base, "_datadog_patch", False):
+        setattr(pynamodb.connection.base, "_datadog_patch", False)
+        unwrap(pynamodb.connection.base.Connection, "_make_api_call")
+
+
+def patched_api_call(original_func, instance, args, kwargs):
+
+    pin = Pin.get_from(instance)
+    if not pin or not pin.enabled():
+        return original_func(*args, **kwargs)
+
+    with pin.tracer.trace(
+        "pynamodb.command", service=config.pynamodb["service_name"], span_type=SpanTypes.HTTP
```

Comment:
```suggestion
        "pynamodb.command", service=trace_utils.ext_service(config.pynamodb, pin, "pynamodb"), span_type=SpanTypes.HTTP
```
you'll need to merge master for this one too

Reply:
The newest version should have this change. I also ended up rebasing the branch for this.